src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -420,6 +449,34 @@ private static void close(final Closeable res) {
             this.output = out;
             this.level = lvl;
         }
+
+        /**
+         * Checks if line is part of a stack trace and should be appended.
+         * @param string String to check
+         * @return boolean Result
+         */
+        private static boolean shouldAppend(final String string) {
```

Comment:
@dean-e-clark can you come up with other name for the variable? `String string` sounds a bit strange ;)